src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -252,7 +267,11 @@ private function getOutputCallback()
      */
     private function getCommandLine($command_args)
     {
-        array_unshift($command_args, $this->command);
+        // Retrieve the trace ID from the TraceId class
+        $trace_id = TraceId::getTraceId();
+
+        // Prepend the trace ID to the command arguments
+        array_unshift($command_args, "TRACE_ID={$trace_id}", $this->command);
```

Comment:
```suggestion
        $env_vars = "TRACE_ID=" . $this->escapeArgument($trace_id}";
        return$env_vars . implode(" ", $this->escapeArguments($command_args));
```
n.b. if you accept this suggestion, you will end up with two "return" statements, because I don't know how to remove lines with this feature (if it's possible).